### ISSUE TITLE
Rebuild all packages with conda >= 2.0?

### ISSUE THREAD

**Issue by stefraynaud** [UV-CDAT/uvcdat#2171]
When trying to build with conda a package that depends on uvcdat,
I faced this error message:

```
WARNING conda_build.build:create_env(697): Build prefix failed with prefix length 255
WARNING conda_build.build:create_env(698): Error was:                                
WARNING conda_build.build:create_env(699): Placeholder of length '80' too short in package uvcdat::binaryio-2.8-np111py27_2.
The package must be rebuilt with conda-build > 2.0.                                                                         
WARNING conda_build.build:create_env(700): One or more of your package dependencies needs to be rebuilt with a longer prefix length.
WARNING conda_build.build:create_env(702): Falling back to legacy prefix length of 80 characters.                                   
WARNING conda_build.build:create_env(703): Your package will not install into prefixes > 80 characters.                             
```

It seems that a lot of uvcdat subpackages have been built a long time ago, including binaryio: https://anaconda.org/uvcdat/repo?page=1

**Comment by doutriaux1** [comment]
@stefraynaud we will be rebuilding all this for 2.10 coming very soon. Thx for reporting!

**Comment by doutriaux1** [comment]
fixed in 2.10
